ui_curses.c

```diff
@@ -921,13 +922,17 @@ static void update_window(struct window *win, int x, int y, int w, const char *t
 {
 	struct iter iter;
 	int nr_rows;
-	int i;
+	int c, i;
 
 	win->changed = 0;
 
 	bkgdset(pairs[CURSED_WIN_TITLE]);
-	sprintf(print_buffer, " %-*s ", w - 2, title);
+	c = snprintf(print_buffer, sizeof(print_buffer), " %s", title);
+	int pad_sz = w - 1 - u_str_width(title);
+	memset(print_buffer + c, ' ', pad_sz);
+	print_buffer[c + pad_sz] = '\0';
```

Comment:
This small fix is not strictly related to the buffer size changes, but I realized sprintf is not multibyte aware (duh) so it won't always add enough padding in the title of the browser tab.

Reply:
Actually, `title` will already have been padded by `format_str()`. Let me just remove the confusing padding specifier here.